src/components/_global/BalCheckbox/BalCheckbox.vue

```diff
@@ -89,7 +90,7 @@ export default defineComponent({
         case 'lg':
           return 'w-6 h-6';
         default:
-          return 'w-5 h-5 mt-1';
```

Comment:
This `mt-1` and the `relative top-2` above in the diff were probably required to make the items-start look right. Please conditionally add these in if the alignment is to the top.